### PR TITLE
CodeQL failing changes

### LINKED ISSUE #1495: CodeQL failing
cc @mstfbl

CodeQL is failing due to the old version of PyTorch. For testing we should be using nightly.

https://github.com/pytorch/audio/blob/723e9a52ebde0afd542b1cc8588598ad2c893c87/.github/workflows/codeql.yml#L34

(Also Ninja should be installed along side of `cmake` via `pip`, not from `apt`)

### PR DESCRIPTION
**Pull request by krishnakalyan3**
Fixes #1495

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -20,19 +20,13 @@ jobs:
...
       - name: Update submodules
         run: git submodule update --init --recursive
 
       - name: Install Torch
         run: |
-              python -m pip install cmake
-              python -m pip install torch==1.8.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
-              sudo ln -s /usr/bin/ninja /usr/bin/ninja-build
+              python -m pip install cmake ninja
+              python -m pip install -f ttps://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```
Comment (mthrok) [inline .github/workflows/codeql.yml:29]:
```suggestion
              python -m pip install  --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

**Review by mthrok — COMMENTED** [review]
Hi @krishnakalyan3 

Thanks for picking up the issue. Please check my comment for fix.

**Comment by krishnakalyan3** [comment]
@mthrok Thanks for the review.

**Review by mthrok — APPROVED** [review]
Looks good. CodeQL is passing. Please mark it 'Ready for review` and we can merge this.

**Comment by mthrok** [comment]
Thanks!

**Pull request merged**